code/modules/mob/living/carbon/emote.dm

```diff
@@ -134,6 +134,23 @@
 		qdel(N)
 		to_chat(user, span_warning("You're incapable of slapping in your current state."))
 
+/datum/emote/living/carbon/shoesteal
+	key = "shoesteal"
+	key_third_person = "shoesteals"
+	hands_use_check = TRUE
+	cooldown = 3 SECONDS
+
+/datum/emote/living/carbon/shoesteal/run_emote(mob/user, params, type_override, intentional)
+	. = ..()
+	if (!.)
+		return
+	var/obj/item/hand_item/stealer/stealing_hand = new(user)
+	if (user.put_in_hands(stealing_hand))
+		user.balloon_alert(user, "preparing to steal shoes")
+	else
+		qdel(stealing_hand)
+		user.balloon_alert(user, "you can't prepare to steal shoes!")
```

Comment:
These balloon alerts are really odd, I don't know if they even need to exist since you clearly get an item in your hand to mark it worked or not, but even despite that, these balloon alerts are way too long, and the first one should end in ``...``

Reply:
Yes they need to exist because the chat box is in the center of your screen, and your hands are at the bottom

Objective is to keep your eyes from moving as much as possible